docs/buildah-umount.md

```diff
@@ -9,6 +9,11 @@ buildah\-umount - Unmount the root file system on the specified working containe
 ## DESCRIPTION
 Unmounts the root file system on the specified working containers.
 
+## OPTIONS
+**--all, -a**
+
+All of the currently mounted containers will be unmounted.
+
```

Comment:
I'd add the example 'buildah umount --all' here, just don't include the output from the command too.